internal/adapter/lsp/document.go

```diff
@@ -259,10 +313,17 @@ func (d *document) DocumentLinks() ([]documentLink, error) {
 		}
 
 		for _, match := range wikiLinkRegex.FindAllStringSubmatchIndex(line, -1) {
+			// Ignore when inside backticks: `[[filename]]`
+			if linkWithinInlineCode(line, match[0], match[1], insideInline) {
+				continue
+			}
 			href := line[match[2]:match[3]]
 			hasTitle := match[4] != -1
 			appendLink(href, match[0], match[1], hasTitle, true)
 		}
+		if strings.Count(line, "`") % 2 == 1 {
```

Comment:
@Rahlir I'm unsure what this `strings.Count()` check is doing? Is it perhaps left over from an initial solution? Because in addition to `linkWithinInlineCode()` it doesn't achieve anything or?